components/ama-rsvp.js

```diff
@@ -0,0 +1,91 @@
+import { useState, useEffect } from 'react'
+import {
+  Button,
+  Box,
+  Card,
+  Grid,
+  Heading,
+  Input,
+  Label,
+  Text,
+  Select,
+  Checkbox
+} from 'theme-ui'
+import fetch from 'isomorphic-unfetch'
+
+const AMARsvp = ({ id }) => {
+  const [email, setEmail] = useState('')
+  const [name, setName] = useState('')
+  const [association, setAssociation] = useState('')
+  const [status, setStatus] = useState('')
+  useEffect(() => {
+    setTimeout(() => {
+      setEmail('')
+      setName('')
+      setAssociation('')
+    }, 2000)
+  }, [status])
+  return (
+    <Card sx={{ mt: [3, 4] }}>
+      <Heading variant="headline" sx={{ mt: 0, mb: 1 }}>
+        RSVP for this AMA
+      </Heading>
+      <form
+        action={`/api/rsvp?id=${id}`}
+        onSubmit={e => {
+          e.preventDefault()
+          fetch(`/api/rsvp?id=${id}`, {
+            method: 'POST',
+            data: JSON.stringify({ phone })
+          })
+            .then(r => r.json())
+            .then(r => setStatus(r.status))
+        }}
+      >
+        <Grid gap={3} columns="1fr 1fr" sx={{ alignItems: 'end', mb: 2 }}>
+          <div>
+            <Label htmlFor="name">Name</Label>
+            <Input
+              type="text"
+              id="phone"
+              name="name"
+              placeholder="Hacker McHackerston"
+              sx={{ bg: 'sunken' }}
+            />
+          </div>
+          <div>
+            <Label htmlFor="email">Email</Label>
+            <Input
+              type="email"
+              id="email"
+              name="email"
+              placeholder="mchackerston@hacker.org"
+              sx={{ bg: 'sunken' }}
+            />
+          </div>
+        </Grid>
+        <Box sx={{ mb: 3 }}>
+          <Label>How are you primarily associated with Hack Club?</Label>
```

Comment:
This is an accessibility issue—the `Select` should be inside the `Label`, otherwise the label doesn’t label anything